packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -130,8 +130,8 @@ async fn create_entity<P: GraphPool>(
 async fn get_entity<P: GraphPool>(
```

Comment:
Comment for line 120:
`body = PersistedEntity` instead. We may need to put `Component`.

Reply:
Should be fixed